apps/wappush/js/si_sl_screen_helper.js

```diff
@@ -47,13 +44,13 @@ var SiSlScreenHelper = (function() {
   }
 
   function sssh_populateScreen(message) {
-    var _ = navigator.mozL10n.get;
-
     screen.hidden = false;
 
     // Populate the message
     if (message && !message.isExpired()) {
-      title.textContent = message.sender;
+      var l10nTitle = Utils.prepareMessageTitle(message);
```

Comment:
So here now we'll have `(SIM x)` written if we're in a DSDS environment. Is it what we want ? Maybe yes, I just don't know if this is intended :)